app/controllers/posts_controller.rb

```diff
@@ -12,6 +12,10 @@ def index
     @posts = Post.all
   end
 
+  def show
+    @posts = Post.all
```

Comment:
Could you explain this?

Reply:
We don't need this anymore. the @posts = Post.all will be in the index view.